src/modinfodialogconflicts.cpp

```diff
@@ -723,6 +735,68 @@ ConflictItem GeneralConflictsTab::createOverwrittenItem(
     index, std::move(fileName), true, std::move(altOrigin), archive);
 }
 
+QString percent(int a, int b) {
+  if (b == 0) {
+    return QString::number(0, 'f', 2);
+  }
+  return QString::number((((float)a / (float)b) * 100), 'f', 2);
+}
+
+void GeneralConflictsTab::updateUICounters()
+{
+  ui->overwriteCount->display(m_counts.numOverwrite);
+  ui->overwrittenCount->display(m_counts.numOverwritten);
+  ui->noConflictCount->display(m_counts.numNonConflicting);
+
+  QString tooltipBase = tr("<table cellspacing=\"5\">"
```

Comment:
I don't really like "loose" and "losing" being in the same table, but I don't really have an alternative.

Reply:
I know, but I couldn't figure out anything else either